### PR TITLE
Added move, move_ip, type stubs, renamed params

### PR DESCRIPTION
**Pull request by itzpr3d4t0r**
The polygon type was missing typestubs for the move functions. I also took a quick look at pygame naming scheme for move functions and renamed the parameters to align with them.

### CODE REVIEW

File: geometry.pyi [hunk 1]
```diff
@@ -20,7 +20,7 @@ Coordinate = Union[Tuple[float, float], Sequence[float], Vector2]
 
 _CanBeLine = Union[
     Rect,
-    "Line",
+    Line,
```
Comment (Emc2356) [inline geometry.pyi:23]: nope cause the `Line` object is not defined yet

File: geometry.pyi [hunk 2]
```diff
@@ -34,8 +34,8 @@ class _HasLineAttribute(Protocol):
 
 LineValue = Union[_CanBeLine, _HasLineAttribute]
 
-_CanBeCircle = Union[Vector3, "Circle", Tuple[float, float, float], Sequence[float]]
-_CanBeCollided = Union[Circle, Rect, Sequence[int, int]]
+_CanBeCircle = Union[Vector3, Circle, Tuple[float, float, float], Sequence[float]]
+_CanBeCollided = Union[Circle, Rect, Line, Sequence[int, int]]
```
Comment (Emc2356) [inline geometry.pyi:38]: same here

File: geometry.pyi [hunk 3]
```diff
@@ -73,12 +74,12 @@ class Line(Sequence[float]):
     def __setitem__(self, key: int, value: float) -> None: ...
     @overload
     def __setitem__(self, key: slice, value: Union[float, Line]) -> None: ...
-    def __copy__(self) -> "Line": ...
+    def __copy__(self) -> Line: ...
```
Comment (Emc2356) [inline geometry.pyi:77]: in the signatures of the class the class is not defined yet so same here

File: geometry.pyi [hunk 4]
```diff
@@ -104,19 +105,19 @@ class Line(Sequence[float]):
     @overload
     def collidecircle(self, x: float, y: float, r: float) -> bool: ...
     def raycast(
-        self, sequence: Sequence[Union["Line", Circle]]
+        self, sequence: Sequence[Union[Line, Circle, Rect]]
```
Comment (Emc2356) [inline geometry.pyi:108]: same

File: geometry.pyi [hunk 5]
```diff
@@ -104,19 +105,19 @@ class Line(Sequence[float]):
...
     ) -> Optional[Tuple[float, float]]: ...
     def as_rect(self) -> Rect: ...
     @overload
-    def move(self, x: float, y: float) -> "Line": ...
+    def move(self, x: float, y: float) -> Line: ...
     @overload
-    def move(self, x_y: Coordinate) -> "Line": ...
+    def move(self, move_by: Coordinate) -> Line: ...
```
Comment (Emc2356) [inline geometry.pyi:114]: also here

File: geometry.pyi [hunk 6]
```diff
@@ -156,25 +157,28 @@ class Circle:
     @overload
     def colliderect(self, x: int, y: int, w: int, h: int) -> bool: ...
     def collideswith(self, other: _CanBeCollided) -> bool: ...
-    def __copy__(self) -> "Circle": ...
+    def __copy__(self) -> Circle: ...
```
Comment (Emc2356) [inline geometry.pyi:160]: this is painful

File: geometry.pyi [hunk 7]
```diff
@@ -188,8 +192,16 @@ class Polygon:
     def __init__(self, *args) -> None: ...
     @overload
     def __init__(self, polygon: Polygon) -> None: ...
-    def __copy__(self) -> "Polygon": ...
+    def __copy__(self) -> Polygon: ...
```
Comment (Emc2356) [inline geometry.pyi:195]: i am so sorry

**Review by Emc2356 — COMMENTED** [review]
sorry but i had to mention them

File: geometry.pyi (hunk 1 above)
Comment (itzpr3d4t0r) [reply geometry.pyi:23]: I mean idk i just copied what pygame has and my IDE is not complaining

**Pull request merged**